deploy/gcloud/README.md

```diff
@@ -118,6 +118,60 @@ The following information can be customized in setup.sh, but the default also wo
 
   kubectl apply -f amppackager_service.yaml
 
+## Production Reverse Proxy Setup
+
+For now, productionizing is a bit manual. The minimum steps are:
+
+  1. Don't expose `amppkg` to the outside world; keep it on your internal
+     network. Use the loadBalancerSourceRanges above so that it's only visible
+     to the reverse proxy you setup. More information could be found here:
+     1. [Kubernetes Firewall Configuration](https://kubernetes.io/docs/tasks/access-application-cluster/configure-cloud-provider-firewall/)
+     2. [Subnet Calculator](https://mxtoolbox.com/subnetcalculator.aspx)
+     3. [Understanding CIDR notation](https://www.digitalocean.com/community/tutorials/understanding-ip-addresses-subnets-and-cidr-notation-for-networking)
+  2. Configure your TLS-serving frontend server to conditionally proxy to
+     `amppkg`:
+     1. If the URL starts with `/amppkg/`, forward the request unmodified.
+     2. If the URL points to an AMP page and the `AMP-Cache-Transform` request
+        header is present, rewrite the URL by prepending `/priv/doc` and forward
+        the request.
+
+        NOTE: If using nginx, prefer using `proxy_pass` with `$request_uri`,
+        rather than using `rewrite`, as in [this PR](https://github.com/Warashi/try-amppackager/pull/3),
+        to avoid percent-encoding issues.
+     3. If at all possible, don't send URLs of non-AMP pages to `amppkg`; its
+        [transforms](transformer/) may break non-AMP HTML.
+     4. DO NOT forward `/priv/doc` requests; these URLs are meant to be
+        generated by the frontend server only.
+  3. For HTTP compliance, ensure the `Vary` header set to `AMP-Cache-Transform,
+     Accept` for all URLs that point to an AMP page, irrespective of whether the
+     response is HTML or SXG. (SXG responses that come from `amppkg` will have
+     the appropriate `Vary` header set, so it may only be necessary to
+     explicitly set the `Vary` header for HTML responses.)
+  4. Get an SXG cert from your CA. It must use an EC key with the prime256v1
+     algorithm, and it must have a [CanSignHttpExchanges
+     extension](https://wicg.github.io/webpackage/draft-yasskin-httpbis-origin-signed-exchanges-impl.html#cross-origin-cert-req).
+     One provider of SXG certs is [DigiCert](https://www.digicert.com/account/ietf/http-signed-exchange.php).
+     You can fill in your ACME Directory URL using these [steps](https://www.google.com/url?q=https://docs.digicert.com/certificate-tools/Certificate-lifecycle-automation-index/acme-user-guide/acme-directory-urls-signed-http-exchange-certificates/&sa=D&ust=1605040563702000&usg=AOvVaw3rcv5Q8H8_Js_TrKHmy8rs).
+  5. Every 90 days or sooner, renew your SXG cert (per
+     [WICG/webpackage#383](https://github.com/WICG/webpackage/pull/383)) and
+     restart amppkg (per
+     [#93](https://github.com/ampproject/amppackager/issues/93)).
```

Comment:
Mention this is only necessary if ACME is not enabled.

Reply:
Done.